platforms/quorum/configuration/roles/setup/bootnode/tasks/main.yaml

```diff
@@ -5,31 +5,24 @@
 ##############################################################################################
 
 # This task checks if the bootnode binary is already in place or not
-- name: Check bootnode
+- name: Check bootnode 
   stat:
     path: "{{ bin_install_dir }}/bootnode"
   register: bootnode_stat_result
 
-# This task checks if quorum repo directory exists
-- name: Check quorum repo dir exists
-  stat:
-    path: "{{ bin_install_dir }}/quorum"
-  register: quorum_stat_result
-
-# This task clones the quorum git repository
-- name: Clone the git repo
-  git:
-    repo: "{{ bootnode.repo }}"
-    version: master
-    force: yes
-    dest: "{{ bin_install_dir }}/quorum"
-  when: not ( quorum_stat_result.stat.exists and bootnode_stat_result.stat.exists )
-
-# This task builds the bootnode binary
-- name: Make bootnode
-  make:
-    chdir: "{{ bin_install_dir }}/quorum"
-    target: bootnode
+# This task creates the bootnode directory
+- name: Create bootnode directory
+  file:
+    path: "{{ bin_install_dir }}/bootnode"
+    state: directory
+  when: bootnode_stat_result.stat.exists == False
+  
+# This task downloads the bootnode tar 
+- name: Download bootnode tar 
+  get_url:
+    url: "https://gethstore.blob.core.windows.net/builds/geth-alltools-linux-amd64-1.10.0-56dec25a.tar.gz"
```

Comment:
Placing the url as the variable under the vars folder (Similar to how it was done with the `bootnode.repo` var) is better here (So that you can quickly edit the vars section instead of searching for the url in this tasks file and keep code organized)